package.json

```diff
@@ -32,6 +31,7 @@
     "@storybook/react": "^4.0.0",
     "@types/faker": "^4.1.4",
     "@types/jest": "^23.3.2",
+    "@types/lodash.throttle": "^4.1.4",
```

Comment:
Sorry if I wasn't clear, but it was a good idea to keep this in dependencies. The reason is that when another TypeScript user imports UI, our types depend on `@types/lodash.throttle` since `lodash.throttle` is a dependency, so it's recommended to include it as a `dependency` even though it's for a dev tool.